types/node/events.d.ts

```diff
@@ -130,10 +135,20 @@ declare module "events" {
      * It supports the following option:
      * @since v0.1.26
      */
-    class EventEmitter<T extends EventMap<T> = DefaultEventMap> {
+    class EventEmitter<Events extends EventMap<Events> = {}> {
         constructor(options?: EventEmitterOptions);
 
-        [EventEmitter.captureRejectionSymbol]?<K>(error: Error, event: Key<K, T>, ...args: Args<K, T>): void;
+        // This "property" is used to brand a specific instance of the EventEmitter class with its Event map, which is needed
+        // in order to infer the map if we have a chain like `class A extends EventEmitter<{}>` (or many levels deep)
+        // It is also marked as possibly undefined in order to allow something like `const t: NodeJS.EventEmitter<{}> = { <insert implementation here> };`
+        readonly #internalTypeOnlyBrand?: Events;
```

Comment:
For what it's worth, the opposite is true here.

The module-exported `EventEmitter` class imports definitions from the `NodeJS.EventEmitter` interface, not the other way around. This means that with this changeset, `NodeJS.EventEmitter` is no longer assignable to `events.EventEmitter`, as it's missing the private pseudoproperty ([TSP example](https://www.typescriptlang.org/play/?#code/JYWwDg9gTgLgBAcgHYQCYFMBc6Bu6kwDOCA3AFBkD0lcAssEqAIYA2cU6YUEANHAEZNC6VHAhI4AYgBsATnkB2MhgDGLJhzhqhhOAFE8BPSGAwY6KAB4D+GLSZg46AB7mkqXQCV0K6KksA1ugAnhAAZvqGdg58AK5IASgA7kgA2gC6AHxwALxwAN4Avtn5ZHDl7OhMqOIswVJcEGAWMMGYcExIweQVcIQwTDDAKnAg6DAAFmgAFOgmZhbtNkbz5lAAlO04EMCo5IVkDGthTCrokbbGpmvWUfaOLm4ecN6+UP5BoRHL0WBxCck0llcgVik5XPhngA5NDoABSAGUAHQ-K4LKw-e4lA5UGgAEVi5xgEDgk3OJyCcC4wBwg3OjWasGCfBhGERKKiaLWcGAuhQcBY4gA5hYOoRCMAhUgmPwWESSajVhZlD51JpfEh+lpYv0ICAuYs4Kz4cjFdcLJYiplyGb0UixpMZiodcT9UqNuQgA))

The overall module cleanup in this PR should definitely be kept, but I really think that the best course of action is to keep the top-level `once()` and `on()` as non-generic and "dumb-typed".